products/sle/main.pm

```diff
@@ -380,7 +380,10 @@ sub load_x11regression_documentation {
         loadtest "x11regressions/libreoffice/libreoffice_mainmenu_components";
         loadtest "x11regressions/libreoffice/libreoffice_recent_documents";
         loadtest "x11regressions/libreoffice/libreoffice_default_theme";
-        loadtest "x11regressions/libreoffice/libreoffice_pyuno_bridge";
+        loadtest "x11regressions/evolution/evolution_prepare_servers";
+        if (get_var("VERSION") =~ /12/) {
```

Comment:
@okurz It is fixed now, also for the other test which used same logic (which is not necessary).